src/node_crypto.cc

```diff
@@ -642,7 +654,7 @@ void SecureContext::AddCACert(const FunctionCallbackInfo<Value>& args) {
   (void) &clear_error_on_return;  // Silence compiler warning.
 
   if (args.Length() != 1) {
-    return env->ThrowTypeError("Bad parameter");
+    return env->ThrowTypeError("CA Certificate argument is mandatory");
```

Comment:
I'd write it as `"CA certificate", i.e. without capitalizing the second word.  Applies to some of the other messages too.